draft-packages/illustration/KaizenDraft/Illustration/Base.module.scss

```diff
@@ -1,8 +1,40 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+
 .wrapper {
   width: 100%;
   margin: 0;
 }
 
+.figure {
+  margin: 0;
+  position: relative;
+
+  // nested to get more specificity, beat out the generic button styles
+  .pausePlayButton {
+    position: absolute;
+    right: 1rem;
+    bottom: 1rem;
+    /* stylelint-disable declaration-no-important */
+    background-color: $color-white !important;
+    border: 1px solid $color-gray-400;
```

Comment:
This border is a 1.5:1 ratio against white. We can try to get away with it if we want, I'm a bit hazy about whether the icon by itself is enough here.